### ISSUE TITLE
When Sideshow can't find subject, it stops whole Wizard, blocks the UI

### ISSUE THREAD

**Issue by DanielApt** [SideshowJS/Sideshow#16]
When Sideshow can't find a subject, it throws the error `Uncaught SSException: [SIDESHOW_E#00000100] Invalid subject.`

This then closes the step description, but not the wizard. With the close button on the step description, there is no obvious way to exit the wizard, except by pressing the *Escape key*.

This needs to be addressed.

**Comment by alcidesqueiroz** [comment]
@DanielApt indeed, there's no reason for giving a higher precedence to `beforeStep` event than the `skipIf` evaluator. I make this change and pushed to develop: bc5f9ef.

**Comment by alcidesqueiroz** [comment]
Closed the wrong issue last night... reopening.=)
